ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -9338,7 +9338,7 @@ static int zend_jit_do_fcall(dasm_State **Dst, const zend_op *opline, const zend
 			}
 
 			if (call_num_args <= func->op_array.num_args) {
-				if (!trace || (trace->op == ZEND_JIT_TRACE_END
+				if (!trace || ZEND_OBSERVER_ENABLED || (trace->op == ZEND_JIT_TRACE_END
```

Comment:
No. This is completely wrong. You don't need to calculate and load opline into register.

In tracing JIT, we often inline the function calls and the first opline of the inlined function is know.
It's must be recorded in the trace. Probably it should be in the next trace ``ZEND_JIT_TRACE_VM`` record.
So for inlined trace functions weshould load the EX(opline) by ``SET_EX_OPLINE opline`` instead of ``SAVE_IP``.